src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useState } from 'react';
+import debounce from 'lodash.debounce';
+
+import { Person } from '../types/Person';
+import classNames from 'classnames';
+
+type Props = {
+  delay: number;
+  people: Person[];
+  onPersonSelect: (person: Person | null) => void;
+};
+
+export const Autocomplete: React.FC<Props> = ({
+  delay,
+  people,
+  onPersonSelect = () => {},
+}) => {
+  const [query, setQuery] = useState('');
+  const [suggestedPeople, setSuggestedPeople] = useState(people);
+  const [isListVisible, setIsListVisible] = useState(false);
+
+  const debounceSuggestedPeople = useCallback(
+    debounce(setSuggestedPeople, delay),
+    [],
+  );
```

Comment:
Fix this linter error 
<img width="881" alt="image" src="https://github.com/user-attachments/assets/3b73c2e8-b4b6-4a38-9ca4-bf01a882d7cf">